regions_www/m/pct_shiny_funs.R

```diff
@@ -589,8 +607,14 @@ popup_zones <- function(data, scenario, purpose){
     </tr>
     <tr>
       <td> Change in deaths/yr: &nbsp; </td>
-      <td style= 'color:", font_colour , "' >", round(data[[data_filter(scenario, "sideath_heat")]], 3),
-             " (&pound;" , round(data[[data_filter(scenario, "sivalue_heat")]]), ")
+        <td style= 'color:", font_colour_death , "' >", round(data[[data_filter(scenario, "sideath")]], 3),
```

Comment:
We are using the colour based on `sivalueyll` but we are displaying `sideath` (and `sivalueyll`).  I'm guessing they are pretty much the same but wanted to check.

Reply:
yes thanks, when one is negative the other is almost always positive...and in the 0.2% where that is not true (and both are v close to zero) makes sense to me to choose just one colour, and on balance the sivalueyll is the better one.

Reply:
The reason they could be different is that deaths are weighted equally, whereas (YLLs are weighted according to the age of the person. So if you got three extra deaths among older people but two were averted among younger people then you could possibly get the two numbers pointing in different directions. It is rare)